test/System.IdentityModel.Tokens.Jwt.Tests/JwtSecurityTokenHandlerTests.cs

```diff
@@ -1356,7 +1356,7 @@ public static TheoryData<JWEDecompressionTheoryData> JweDecompressSizeTheoryData
 
             TheoryData<JWEDecompressionTheoryData> theoryData = new TheoryData<JWEDecompressionTheoryData>();
 
-#if NETCOREAPP2_1
+#if NET2_1
```

Comment:
Is this a valid flag? I don't see it here: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/preprocessor-directives#conditional-compilation